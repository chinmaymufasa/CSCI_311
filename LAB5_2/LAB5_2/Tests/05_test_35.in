2
9
2
197
2
10
2
120
2
108
2
189
2
31
2
73
2
158
2
166
2
188
2
135
2
82
2
83
2
55
2
90
2
27
2
88
2
177
2
145
4
5
6
1
135
-1
