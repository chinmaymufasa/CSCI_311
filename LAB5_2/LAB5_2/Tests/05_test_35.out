120 73 10 9 31 27 55 83 82 90 88 108 166 145 135 158 189 188 177 197 
9 10 27 31 55 73 82 83 88 90 108 120 135 145 158 166 177 188 189 197 
9 27 55 31 10 82 88 108 90 83 73 135 158 145 177 188 197 189 166 120 
135
